include/ultra64/libc.h

```diff
@@ -5,8 +5,14 @@
 
 void osSyncPrintf(const char* fmt, ...);
 
+#ifdef __GNUC__
+void bzero(void* __s, unsigned int __n);
+int bcmp(const void* __sl, const void* __s2, unsigned int __n);
+void bcopy(const void* __src, void* __dest, unsigned int __n);
+#else
```

Comment:
Why?

Reply:
gcc recognizes these as builtins and warns because they have the wrong prototype, and since this is in a header the warning repeats for all files 💀
this fixes that warning (and avoids the spam)

Reply:
are the implementations still the same using gcc? does it not error or at least warn that the prototypes are different in the definition

Reply:
I'm not sure what you're asking exactly; the implementations for these are assembly and are still called by gcc, but gcc has prototypes for these registered internally and doesn't agree with the libultra prototypes creating a warning

Reply:
oh the implementations are assembly, I forgot that point. okay then nvm